server/controllers/recall/recallDateController.ts

```diff
@@ -29,6 +30,23 @@ export default class RecallDateController extends RecallBaseController {
         validationErrors.recallDate = this.formError('recallDate', 'mustBeAfterEarliestSentenceDate')
       }
 
+      const existingAdjustments: AdjustmentDto[] = getExistingAdjustments(req)
+
+      const recallDate = new Date(values.recallDate as string)
+
+      const isWithinAdjustment = existingAdjustments.some(adjustment => {
+        if (!adjustment.fromDate || !adjustment.toDate) return false
+
+        const fromDate = new Date(adjustment.fromDate)
```

Comment:
[date-fns ](https://date-fns.org/v4.1.0/docs/isBefore) allows you to give it strings so we don't 100% need the new Date()